modules/pool/invest/components/PoolInvestProportional.tsx

```diff
@@ -100,10 +106,7 @@ export function PoolInvestProportional({ onShowPreview }: Props) {
                                     <Box flex="1">
                                         <TokenSelectInline
                                             tokenOptions={option.tokenOptions}
-                                            selectedAddress={
-                                                selectedOptions[`${option.poolTokenIndex}`] ||
-                                                option.tokenOptions[0].address
-                                            }
+                                            selectedAddress={selectedOptions[`${option.poolTokenIndex}`]}
```

Comment:
this would not be guaranteed to be defined on the first render, not sure if that would cause issues in the component